website/sidebars.js

```diff
@@ -103,6 +103,31 @@ module.exports = {
         },
       ],
     },
+    {
+      type: "category",
+      label: "Elixir SDK (Experimental)",
```

Comment:
@crjm I did as you suggested and am wondering if there is a way to make this look nicer?

Reply:
While it doesn't look great, it will have to do for now:

<img width="1440" alt="image" src="https://github.com/dagger/dagger/assets/3342/dcb5cbba-656d-49bd-ab28-50c9215614e3">